scripts/ci/package-unica-runtime.py

```
@@ -222,11 +266,45 @@ def load_bundle(bundle_root: Path) -> tuple[dict, list[tuple[PurePosixPath, Path
     manifest_bytes = (
         json.dumps(generated_manifest, ensure_ascii=False, indent=2, sort_keys=True) + "\n"
     ).encode("utf-8")
+    # Версия артефакта берётся у инструментов, которые он несёт: они делят
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `у` (CYRILLIC SMALL LETTER U). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF003)